### PR TITLE
Added support of PDF report generation for cases

### PR DESCRIPTION
**Pull request by Wolf-Legend**
Added support in CLI tool for generating PDF for extracted data session.
Example :
Data Extraction
``` 
python3 collector.py -o all -sn case-07-james-bond 
```
Report Generation
```
 python3 collector.py -rp case-07-james-bond 
```

Sample Report generated from emulator device
[report.pdf](https://github.com/scorelab/OpenMF/files/3523528/report.pdf)

Report contains following details if run on a actual rooted device

- ADB Serial
- Manufacturer
- Model
- IMEI 
- Android Version
- Build Name
- WiFi Mac
- Local Time
- Android Time
- Current Sim Serial Number
- Current Sim Phone Number
- Current Sim Operator Name
- Previous Sim Serial Number
- Previous Sim Phone Number
- Google Accounts

### CODE REVIEW

**Comment by Wolf-Legend** [comment]
@charithccmc @vivonk PDF feature is done, now moving to web version part :) 
Please review this one

File: collector.py [hunk 1]
```diff
@@ -71,13 +79,23 @@ def collect_data(extract_options, current_session_name):
...
+
+
 if __name__ == '__main__':
     args = sys.argv
     args_set = set(args)
 
     if HELP_KEYS & args_set:
         print(help_str)
         sys.exit()
+    elif REPORT_GEN_KEYS & args_set:
```
Comment (vivonk) [inline collector.py:96]: Can't we generate the report, along with data extraction? @Wolf-Legend

File: data_store/db_helper.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+statements = {
```
Comment (vivonk) [inline data_store/db_helper.py:1]: Is this useful somewhere in this feature or will be used afterward?

File: requirements.txt [hunk 3]
```diff
@@ -0,0 +1,2 @@
+Flask==1.1.1
+pdfkit==0.6.1
```
Comment (vivonk) [inline requirements.txt:2]: Nice Choice :) @Wolf-Legend

File: scripts/io_helper.py [hunk 4]
```diff
@@ -0,0 +1,4 @@
+def write_to_file(filepath, content):
```
Comment (vivonk) [inline scripts/io_helper.py:1]: Returning a boolean flag on status of writing file would be great

File: collector.py (hunk 1 above)
Comment (Wolf-Legend) [reply collector.py:96]: For simplicity, I decided both feature to be separate. But we can add a flag in data extraction for generating report as well along side.

File: data_store/db_helper.py (hunk 2 above)
Comment (Wolf-Legend) [reply data_store/db_helper.py:1]: This is for web version part, will be used later

File: scripts/io_helper.py (hunk 4 above)
Comment (Wolf-Legend) [reply scripts/io_helper.py:1]: Sure bro :)

Comment (vivonk) [reply scripts/io_helper.py]: Done @vivonk

Comment (Wolf-Legend) [reply scripts/io_helper.py:1]: Done @vivonk

Comment (Wolf-Legend) [reply scripts/io_helper.py:1]: Can you review it again? @vivonk

**Review by vivonk — APPROVED** [review]
LGTM, Great Work @Wolf-Legend

**Review by charithccmc — APPROVED** [review]
LGTM 👍

**Pull request merged**